nmrfx-analyst-gui/src/main/java/org/nmrfx/analyst/gui/datasetbrowser/DatasetBrowserTableView.java

```diff
@@ -1,12 +1,22 @@
 package org.nmrfx.analyst.gui.datasetbrowser;
 
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.collections.transformation.FilteredList;
 import javafx.scene.control.TableColumn;
 import javafx.scene.control.TableView;
 import javafx.scene.control.cell.PropertyValueFactory;
 import org.controlsfx.control.tableview2.TableView2;
 import org.nmrfx.utilities.DatasetSummary;
 
+import java.util.List;
+
 public class DatasetBrowserTableView extends TableView2<DatasetSummary> {
```

Comment:
Did not end up using FilteredTableView as it seemed to display rows based on matching each column cell with an "AND" relationship instead of the "OR" relationship that we wanted.